### ISSUE TITLE
have an island value of one block depending on the world

### ISSUE THREAD

**Issue by organizatsiya** [BentoBoxWorld/Level#217]
Hello

I tried to do:
```
worlds:
   bskyblock_world_nether:
     OAK_LOG: 8
```
but it doesn't work because I think "worlds" should be called "Game Modes" instead.
It would be interesting to be able to have different values depending on the world in which the block is placed: overworld, nether or end

**Comment by tastybento** [comment]
No, it is worlds. You have to list the world name there, not the game mode.

The code is storing the value for that world, but the counter is not aware of the world the block is in, just the overworld of the island, so that's why it isn't counted. I'll mark this as an enhancement.
